iphone/titanium.xcconfig

```diff
@@ -4,13 +4,13 @@
 // OF YOUR TITANIUM SDK YOU'RE BUILDING FOR
 //
 //
-TITANIUM_SDK_VERSION = 5.5.0.GA
+TITANIUM_SDK_VERSION = 5.5.1.GA
```

Comment:
It's the latest patch SDK, so it's recommended to use that when building the module - this does not change the minimum SDK